### ISSUE TITLE
Better aria labels needed for calendar toggle on Results filter page

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4080]
## Area or Component
Results tab - filters

## Quick Summary
The accessible name for both calendar buttons is "toggle calendar", but they trigger different actions and so should have unique aria names. (One opens the date picker for start date and the other opens the date picker for end date)

## Accessibility Violation
Name: The element's name is missing or incorrect: https://cdc-devtools.dequecloud.com/rule-help/en/aria-name-missing-incorrect

## Action Requested
The two elements should have unique `aria-name` tags.

## Screenshots (optional)
The two calendar icons in the image here are the source of the issue:
![Screen Shot 2022-07-26 at 3 56 50 PM](https://user-images.githubusercontent.com/80282552/181126175-d3d6b3a5-c6bc-4b2e-ad98-835f637cf7af.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1978534823&range=4:5

**Comment by BobanL** [comment]
Since this component is coming from that [uswds library](https://github.com/trussworks/react-uswds/blob/268d4e65aed4d5f9541fb487a4fa5b3ec152b86c/src/components/forms/DatePicker/DatePicker.tsx#L295), that aria label has a hard set value and cannot be changed. I think the best thing to do is just replace the date picker with the standard input field as was discussed about in other tickets.
